ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -839,8 +859,11 @@ func isPIInstanceRefreshFunc(client *st.IBMPIInstanceClient, id, instanceReadySt
 		if err != nil {
 			return nil, "", err
 		}
+		if (*pvm.Status == "SHUTOFF") && (pvm.Health.Status == instanceReadyStatus || pvm.Health.Status == helpers.PIInstanceHealthOk) {
+			return pvm, helpers.PIInstanceAvailable, nil
+		}
 		// Check for `instanceReadyStatus` health status and also the final health status "OK"
-		if *pvm.Status == helpers.PIInstanceAvailable && (pvm.Health.Status == instanceReadyStatus || pvm.Health.Status == helpers.PIInstanceHealthOk) {
+		if (*pvm.Status == helpers.PIInstanceAvailable) && (pvm.Health.Status == instanceReadyStatus || pvm.Health.Status == helpers.PIInstanceHealthOk) {
```

Comment:
Brackets are not needed. Can keep this line as it is.

Reply:
Updated it